tests/baselines/reference/api/typescript.d.ts

```diff
@@ -8721,7 +8747,7 @@ declare namespace ts {
     function isIdentifierStart(ch: number, languageVersion: ScriptTarget | undefined): boolean;
     function isIdentifierPart(ch: number, languageVersion: ScriptTarget | undefined, identifierVariant?: LanguageVariant): boolean;
     function createScanner(languageVersion: ScriptTarget, skipTrivia: boolean, languageVariant?: LanguageVariant, textInitial?: string, onError?: ErrorCallback, start?: number, length?: number): Scanner;
-    type ErrorCallback = (message: DiagnosticMessage, length: number, arg0?: any) => void;
+    type ErrorCallback = (message: DiagnosticMessage, pos: number, length: number, ...args: DiagnosticArguments) => DiagnosticWithDetachedLocation | undefined;
```

Comment:
I just noticed this; yeah we do not want to be breaking this API.

Reply:
In general I don't think this change should contain any API changes at all.

Reply:
OK, I will revert it.